orca-keel/src/test/kotlin/com/netflix/spinnaker/orca/keel/ImportDeliveryConfigTaskTests.kt

```diff
@@ -377,26 +379,32 @@ internal class ImportDeliveryConfigTaskTests : JUnit5Minutests {
                 manifestLocation.manifest,
                 manifestLocation.ref
               )
-            } throws RetrofitError.httpError(
+            } returns manifest
+          }
+          with(keelService){
+            every {
+              publishDeliveryConfig(manifest)
+            } throws SpinnakerHttpException(RetrofitError.httpError(
               "http://keel",
               Response(
                 "http://keel", 403, "", emptyList(),
-                JacksonConverter(objectMapper).toBody(accessDeniedError) as TypedInput
+               converter.toBody(accessDeniedError) as TypedInput
               ),
-              null, null
-            )
+              converter, null
+            ))
           }
         }
 
         test("task fails and includes the error details returned by keel") {
           val result = execute(manifestLocation.toMap())
           expectThat(result.status).isEqualTo(ExecutionStatus.TERMINAL)
-          expectThat(result.context["error"]).isA<SpringHttpError>().isEqualTo(accessDeniedError)
+          expectThat(result.context["error"]).isA<SpringHttpError>()
```

Comment:
```suggestion
          expectThat(result.status).isEqualTo(ExecutionStatus.TERMINAL)
          expectThat(result.context["error"]).isA<SpringHttpError>()
          val keelError: SpringHttpError = result.context["error"] as SpringHttpError
          //before comparing, copy timestamp as they don't match even when all the other details do
          accessDeniedError = accessDeniedError.copy(timestamp = keelError.timestamp)
          expectThat(result.context["error"]).isA<SpringHttpError>().isEqualTo(accessDeniedError)
```

Reply:
I have moved this test scenario to java file which uses wiremock. This is covered in the commit : https://github.com/spinnaker/orca/commit/7a48b311ff9454124496432a02fe12e1db9aae8e